.github/workflows/CODE_SCANNING.yml

```diff
@@ -1,4 +1,4 @@
-name: "Code Scanning"
+name: CODE_SCANNING
```

Comment:
I think changing the name will cause the workflow to appear in [GH actions](https://github.com/bpmn-io/moddle-xml/actions) section two times, one with the old and one with the new name, just FYI

Reply:
I think we're fine: https://github.com/camunda/camunda-modeler/pull/4657#issuecomment-2451970785.